CRITICAL - 2022-02-13 20:15:07 --> Undefined array key "foto"
#0 C:\xampp\htdocs\ci4app\app\Views\News\detail.php(48): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 48)
#1 C:\xampp\htdocs\ci4app\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ci4app\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ci4app\system\Common.php(1089): CodeIgniter\View\View->render('News/detail', Array, true)
#4 C:\xampp\htdocs\ci4app\app\Controllers\News.php(111): view('News/detail', Array)
#5 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\News->detail('prediksi-villai...')
#6 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#7 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-13 20:19:36 --> Cannot use object of type stdClass as array
#0 C:\xampp\htdocs\ci4app\system\View\View.php(210): include()
#1 C:\xampp\htdocs\ci4app\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 C:\xampp\htdocs\ci4app\system\Common.php(1089): CodeIgniter\View\View->render('News/detail', Array, true)
#3 C:\xampp\htdocs\ci4app\app\Controllers\News.php(115): view('News/detail', Array)
#4 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\News->detail('prediksi-villai...')
#5 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#6 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2022-02-13 20:47:03 --> Unknown column 'created_at' in 'field list'
CRITICAL - 2022-02-13 20:47:03 --> Unknown column 'created_at' in 'field list'
#0 C:\xampp\htdocs\ci4app\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 C:\xampp\htdocs\ci4app\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 C:\xampp\htdocs\ci4app\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci4app\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci4app\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\ci4app\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\ci4app\app\Controllers\Category.php(49): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\Category->addCategory()
#10 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
#11 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2022-02-13 20:49:59 --> Unknown column 'created_at' in 'field list'
CRITICAL - 2022-02-13 20:49:59 --> Unknown column 'created_at' in 'field list'
#0 C:\xampp\htdocs\ci4app\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 C:\xampp\htdocs\ci4app\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 C:\xampp\htdocs\ci4app\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci4app\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci4app\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\ci4app\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\ci4app\app\Controllers\Category.php(49): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\Category->addCategory()
#10 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
#11 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2022-02-13 20:54:29 --> Unknown column 'created_at' in 'field list'
CRITICAL - 2022-02-13 20:54:29 --> Unknown column 'created_at' in 'field list'
#0 C:\xampp\htdocs\ci4app\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 C:\xampp\htdocs\ci4app\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 C:\xampp\htdocs\ci4app\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci4app\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci4app\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\ci4app\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\ci4app\app\Controllers\Category.php(46): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\Category->addCategory()
#10 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
#11 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2022-02-13 20:57:07 --> Unknown column 'created_at' in 'field list'
CRITICAL - 2022-02-13 20:57:07 --> Unknown column 'created_at' in 'field list'
#0 C:\xampp\htdocs\ci4app\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 C:\xampp\htdocs\ci4app\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 C:\xampp\htdocs\ci4app\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci4app\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci4app\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\ci4app\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\ci4app\app\Controllers\Category.php(46): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\Category->addCategory()
#10 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
#11 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2022-02-13 21:06:23 --> Unknown column 'created_at' in 'field list'
CRITICAL - 2022-02-13 21:06:23 --> Unknown column 'created_at' in 'field list'
#0 C:\xampp\htdocs\ci4app\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 C:\xampp\htdocs\ci4app\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 C:\xampp\htdocs\ci4app\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci4app\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci4app\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\ci4app\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\ci4app\app\Controllers\Category.php(46): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\Category->addCategory()
#10 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
#11 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2022-02-13 21:07:06 --> Unknown column 'created_at' in 'field list'
CRITICAL - 2022-02-13 21:07:06 --> Unknown column 'created_at' in 'field list'
#0 C:\xampp\htdocs\ci4app\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 C:\xampp\htdocs\ci4app\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 C:\xampp\htdocs\ci4app\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci4app\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci4app\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\ci4app\app\Controllers\Category.php(46): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\Category->addCategory()
#9 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
#10 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2022-02-13 21:08:17 --> There is no data to insert.
#0 C:\xampp\htdocs\ci4app\system\BaseModel.php(1574): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
#1 C:\xampp\htdocs\ci4app\system\BaseModel.php(704): CodeIgniter\BaseModel->transformDataToArray(Array, 'insert')
#2 C:\xampp\htdocs\ci4app\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#3 C:\xampp\htdocs\ci4app\app\Controllers\Category.php(45): CodeIgniter\Model->insert(Array)
#4 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\Category->addCategory()
#5 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
#6 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2022-02-13 21:11:51 --> Unknown column 'created_at' in 'field list'
CRITICAL - 2022-02-13 21:11:51 --> Unknown column 'created_at' in 'field list'
#0 C:\xampp\htdocs\ci4app\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 C:\xampp\htdocs\ci4app\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 C:\xampp\htdocs\ci4app\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ci4app\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ci4app\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\ci4app\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\ci4app\app\Controllers\Category.php(46): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\Category->addCategory()
#10 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
#11 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2022-02-13 21:16:32 --> Unknown column 'nama_kategori' in 'where clause'
CRITICAL - 2022-02-13 21:16:32 --> Unknown column 'nama_kategori' in 'where clause'
#0 C:\xampp\htdocs\ci4app\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\ci4app\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci4app\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4app\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('Games', 'nama_kategori', Array, NULL)
#6 C:\xampp\htdocs\ci4app\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('nama_kategori', 'nama_kategori', 'Games', Array, Array)
#7 C:\xampp\htdocs\ci4app\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ci4app\app\Controllers\Category.php(35): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\Category->addCategory()
#10 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
#11 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2022-02-13 21:17:22 --> Unknown column 'nama_kategori' in 'where clause'
CRITICAL - 2022-02-13 21:17:22 --> Unknown column 'nama_kategori' in 'where clause'
#0 C:\xampp\htdocs\ci4app\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\ci4app\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci4app\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4app\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('Game', 'nama_kategori', Array, NULL)
#6 C:\xampp\htdocs\ci4app\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('nama', 'nama', 'Game', Array, Array)
#7 C:\xampp\htdocs\ci4app\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ci4app\app\Controllers\Category.php(35): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\Category->addCategory()
#10 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
#11 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2022-02-13 21:28:21 --> Unknown column 'nama_kategori' in 'where clause'
CRITICAL - 2022-02-13 21:28:21 --> Unknown column 'nama_kategori' in 'where clause'
#0 C:\xampp\htdocs\ci4app\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\ci4app\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci4app\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4app\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('Game', 'nama_kategori', Array, NULL)
#6 C:\xampp\htdocs\ci4app\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('nama_kat', 'nama_kat', 'Game', Array, Array)
#7 C:\xampp\htdocs\ci4app\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ci4app\app\Controllers\Category.php(35): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\Category->addCategory()
#10 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
#11 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2022-02-13 21:33:12 --> Unknown column 'nama_kategori' in 'where clause'
CRITICAL - 2022-02-13 21:33:12 --> Unknown column 'nama_kategori' in 'where clause'
#0 C:\xampp\htdocs\ci4app\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\ci4app\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci4app\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4app\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('Game', 'nama_kategori', Array, NULL)
#6 C:\xampp\htdocs\ci4app\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('nama_kat', 'nama_kat', 'Game', Array, Array)
#7 C:\xampp\htdocs\ci4app\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ci4app\app\Controllers\Category.php(35): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\Category->addCategory()
#10 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
#11 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2022-02-13 21:34:04 --> Unknown column 'nama_kategori' in 'where clause'
CRITICAL - 2022-02-13 21:34:04 --> Unknown column 'nama_kategori' in 'where clause'
#0 C:\xampp\htdocs\ci4app\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\ci4app\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci4app\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4app\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('Game', 'nama_kategori', Array, NULL)
#6 C:\xampp\htdocs\ci4app\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('nama_kat', 'nama_kat', 'Game', Array, Array)
#7 C:\xampp\htdocs\ci4app\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ci4app\app\Controllers\Category.php(35): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\Category->addCategory()
#10 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
#11 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2022-02-13 21:35:17 --> Unknown column 'nama_kategori' in 'where clause'
CRITICAL - 2022-02-13 21:35:17 --> Unknown column 'nama_kategori' in 'where clause'
#0 C:\xampp\htdocs\ci4app\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\ci4app\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci4app\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4app\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('Game', 'nama_kategori', Array, NULL)
#6 C:\xampp\htdocs\ci4app\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('nama_kat', 'nama_kat', 'Game', Array, Array)
#7 C:\xampp\htdocs\ci4app\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ci4app\app\Controllers\Category.php(35): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\Category->addCategory()
#10 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
#11 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2022-02-13 21:37:42 --> Unknown column 'nama_kategori' in 'where clause'
CRITICAL - 2022-02-13 21:37:42 --> Unknown column 'nama_kategori' in 'where clause'
#0 C:\xampp\htdocs\ci4app\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\ci4app\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\ci4app\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci4app\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4app\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('Game', 'nama_kategori', Array, NULL)
#6 C:\xampp\htdocs\ci4app\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('nama_kat', 'nama_kat', 'Game', Array, Array)
#7 C:\xampp\htdocs\ci4app\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ci4app\app\Controllers\Category.php(35): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\Category->addCategory()
#10 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
#11 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2022-02-13 21:38:38 --> in_array(): Argument #2 ($haystack) must be of type array, string given
#0 C:\xampp\htdocs\ci4app\system\BaseModel.php(1142): in_array('nama_kat', 'nama_kat', true)
#1 C:\xampp\htdocs\ci4app\system\BaseModel.php(713): CodeIgniter\BaseModel->doProtectFields(Array)
#2 C:\xampp\htdocs\ci4app\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, false)
#3 C:\xampp\htdocs\ci4app\system\BaseModel.php(658): CodeIgniter\Model->insert(Array, false)
#4 C:\xampp\htdocs\ci4app\app\Controllers\Category.php(46): CodeIgniter\BaseModel->save(Array)
#5 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\Category->addCategory()
#6 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
#7 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-13 21:47:08 --> Undefined array key "id"
#0 C:\xampp\htdocs\ci4app\app\Views\Category\list-category.php(156): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 156)
#1 C:\xampp\htdocs\ci4app\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ci4app\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\ci4app\system\Common.php(1089): CodeIgniter\View\View->render('Category/list-c...', Array, true)
#4 C:\xampp\htdocs\ci4app\app\Controllers\Category.php(20): view('Category/list-c...', Array)
#5 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\Category->index()
#6 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Category))
#7 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-13 21:48:17 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\ci4app\app\Controllers\News.php(104): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to acces...', 'C:\\xampp\\htdocs...', 104)
#1 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(825): App\Controllers\News->delete('6')
#2 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#3 C:\xampp\htdocs\ci4app\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci4app\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
